### PR TITLE
Add user logins info APIs

### PR DESCRIPTION
**Pull request by kubedo8**

### CODE REVIEW

File: lumeer-remote/src/main/java/io/lumeer/remote/rest/UserLoginsService.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+   @GET
+   @Path("info")
+   public List<OrganizationLoginsInfo> getOrganizationsLoginsInfo(@QueryParam("descending") boolean descending) {
+      return organizationFacade.getOrganizationsLoginsInfo(descending);
+   }
+
+   @GET
+   @Path("delete")
+   public List<OrganizationLoginsInfo> getOrganizationsLoginsInfoToDelete(@QueryParam("descending") boolean descending, @QueryParam("monthsOld") int months) {
```
Comment (marvec) [inline lumeer-remote/src/main/java/io/lumeer/remote/rest/UserLoginsService.java:52]: What happens when the descending parameter is not passed? Doesn't it throw an exception? We typically use `Boolean` to allow null values.

File: api/src/main/java/io/lumeer/api/model/OrganizationLoginsInfo.java [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+ */
+package io.lumeer.api.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Set;
+
+public class OrganizationLoginsInfo  {
```
Comment (marvec) [inline api/src/main/java/io/lumeer/api/model/OrganizationLoginsInfo.java:27]: The list of templates used in the projects is missing...

**Review by marvec — CHANGES_REQUESTED** [review]

File: lumeer-remote/src/main/java/io/lumeer/remote/rest/UserLoginsService.java (hunk 1 above)
Comment (kubedo8) [reply lumeer-remote/src/main/java/io/lumeer/remote/rest/UserLoginsService.java:52]: Default value is `false`, so no exception will be thrown.

File: api/src/main/java/io/lumeer/api/model/OrganizationLoginsInfo.java (hunk 2 above)
Comment (kubedo8) [reply api/src/main/java/io/lumeer/api/model/OrganizationLoginsInfo.java:27]: Where can I find information about used template?

**Pull request merged**